Dockerfile

```diff
@@ -1,49 +1,57 @@
-FROM postgres:14.2-alpine AS base
+FROM alpine:3.14
 
 ENV USER hocs
 ENV USER_ID 1000
 ENV GROUP hocs
-ENV NAME hocs-toolbox
-ENV PGDATA /app/data
+ENV NAME hocs-migration-toolbox
 ENV AWS_CLI_VERSION 1.16.207
 
 WORKDIR /app
 
+RUN apk update && apk add bash
+
 RUN addgroup ${GROUP} && \
-    adduser -u ${USER_ID} -G ${GROUP} -h /app -D ${USER}
-RUN mkdir -p /app/scripts && \
-    chown -R ${USER}:${GROUP} /app
-RUN mkdir -p /app/scripts
+    adduser -u ${USER_ID} -G ${GROUP} -h /app -D ${USER} &&\
```

Comment:
Unsure if you need a space after the `&&` or whether it will interpolate that okay.

Reply:
I've removed the spaces from the affected lines. Docker build works fine.